agent/05_agent_create_cluster.sh

```diff
@@ -67,6 +67,10 @@ function force_mirror_disconnect() {
 }
 
 function enable_assisted_service_ui() {
+  if [ ! -z "${MIRROR_IMAGES}" ]; then
```

Comment:
This potentially could be enabled also for non-disconnected environment. I guess using IP_STACK would be a better match in our case

Reply:
OC_MIRROR does imply disconnected, if we use IP_STACK we could still have a disconnected env with IPv4 which would fail when using GUI.

Reply:
Checking IP_STACK it's also the approach take in other part of the CI to verify if the current environment is disconnected or not, ie https://github.com/openshift/release/blob/0e6a79426cd6aeebc497d5d79fd8029aabd74058/ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh#L31-L40

Reply:
Changed to use IP_STACK

Reply:
May be worth using the inverted logic to include also the dualstack case (v4v6), which is considered disconnected as well as the v6 (so that essentially the GUI will be enabled only for v4 case)?

Reply:
No, v4v6 doesn't necessarily require a disconnected install - see https://github.com/openshift-metal3/dev-scripts/blob/master/network.sh#L121

I do think we're getting away from the purpose of this PR which is not related to the setting of the GUI and I'd recommend we handle that in a separate PR.

Reply:
I'm not fully sure about that, but agree that this increment is good enough to proceed and the dualstack case could be eventually fixed in a later PR if required

Reply:
OK thanks. Yes I'm working on changes for v4v6 now and I'll make sure to test this particular case with IP_STACK=v4v6